src/machines/builder_route/axis.config.js

```diff
@@ -0,0 +1,140 @@
+export const axis_config = {
+	id: "Axis",
+	initial: "CheckingAxis",
+	on: {
+		'SELECTION_CHANGED': {
```

Comment:
Though they're not needed I placed the quotes on our on fields, to distinguish state and event names from `xstate` properties.

Reply:
Yeah, I'd remove them as we use different formats for events and states.

Reply:
I'm not sure I follow. What do you mean by different formats?

Also, I must mention that another advantage is that this way syntax highlighting can be leveraged to distinguish between `xstate` props and our state and event names.

Reply:
We use:
- upper-snake case for events, like `SELECTION_CHANGED`
- upper camel Case for States, like `CheckingAxis`
- lower Case for actions and guards, like `isAxisComplete` or `pushHistory`
hence the format determine the type, without ambiguity.
Since we don't have blanks in items name, we don't need quotes.

Reply:
What do you think about leveraging the difference in syntax highlighting between unquoted and quoted property names? I believe it makes it easier to pick distinguish them on code editors.

Reply:
I don't know what you mean.

Reply:
![image](https://user-images.githubusercontent.com/1522062/90016950-2baef680-dc81-11ea-8dea-70ec9d393a8a.png)

Reply:
Of course it depends on the editor's configuration, but generally many themes do distinguish between quoted and unquoted property names.

Reply:
I personally find the first much more noisy:

**With quotes**:

<img width="464" alt="Screenshot 2020-08-12 at 13 53 12" src="https://user-images.githubusercontent.com/1309648/90017584-a8eb6300-dca3-11ea-8dae-29ae9b4e1aff.png">

**Without quotes**:

<img width="462" alt="Screenshot 2020-08-12 at 13 55 04" src="https://user-images.githubusercontent.com/1309648/90017587-a983f980-dca3-11ea-8eb6-5d7212d1906e.png">